README.md

```diff
@@ -19,16 +19,32 @@ Once you have implemented the `fm` program (and tested it to the best of your ab
 
 ## Preprocessing
 
-*What preprocessing data do you store in files, and how?*
+We decided to make a new file with the same name as the previous file as a prefix, ending with '_prepro.txt' instead of '.fa' to connect the two. This preprocessing file contains a fasta-style header, followed by a line initiated with '#' containing the suffix array sa(x), a line with the dictionary for C after a '+' and finally a line initiated by '@' followed by a dictionary containing the table O.
+
+This format with initiating symbols for each line type makes sure the correct information is retrieved later. The preprocessing file thus condenses the information from an entire fasta entry to an identifying header and 3 additional lines.
 
 ## Insights you may have had while implementing the algorithm
+Python's built in function sorted is surprisingly fast and much easier to implement than to make an entire search function. We therefore used this to build the suffix array.
```

Comment:
That only tells you that you should be programming in C. It's speed mostly come from that (although a Timsort *is* very fast). It still runs in $O(n\log n)$, though.